### DISCUSSION TITLE
Connection Closed Kong and Cloudflare SSL handshake failed errors

### DISCUSSION THREAD

**Discussion by Alexa-Green** [supabase/supabase#22652] [Questions]
Am getting arbitrary `TypeError: error sending request for url (http://kong:8000/rest/v1/users?select=*&id=eq.USER_ID` errors when trying to get data from within a called edge functions endpoint--  `channel closed` and `connection closed before message completed`. It changes with the user IDs each time and how many times it errors out. If it is called 30 times, 4 of them might fail and fail consistently. Other calls to the DB fetching the same user details seem to be okay, so I don't believe it's corrupted user data. Have tried resetting the db a lot of times as well. There are a lot of calls happening at once to the same functions endpoint right in a row-- could this be it?

Not sure if it is related, but at the same point when calling the deployed functions, have been getting `Cloudflare SSL handshake failed` errors, sometimes 500, sometimes 520, and sometimes 400 error code. The endpoint is actually returning an HTML document with that text included, but not an actual error.

Am on the smallest compute level and all usages are well within the limits-- any ideas?

**Comment by rookledookle** [comment]
Getting the same error when trying to insert into DB while running locally 

`Error inserting user static event 251684: {
  message: "TypeError: error sending request for url (http://kong:8000/rest/v1/user_static_events?select=*): con"... 54 more characters,
  details: "TypeError: error sending request for url (http://kong:8000/rest/v1/user_static_events?select=*): con"... 462 more characters,
  hint: "",
  code: ""
}`
